src/psyclone/psyGen.py

```diff
@@ -1076,7 +1076,7 @@ def reference_accesses(self, var_accesses):
         :param var_accesses: VariablesAccessInfo instance that stores the \
             information about variable accesses.
         :type var_accesses: \
-            :py:class:`psyclone.core.access_info.VariablesAccessInfo`
+            :py:class:`psyclone.core.VariablesAccessInfo`
         '''
         super(Kern, self).reference_accesses(var_accesses)
```

Comment:
Since we're here, would you mind modernising the `super()` call?

Reply:
Done. I've fixed a few more warnings that were close by